testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/RegisterPage.java

```diff
@@ -232,7 +239,13 @@ public boolean isCurrent() {
 
     @Override
     public void open() {
-        throw new UnsupportedOperationException();
+        oauth.openRegistrationForm();
+        assertCurrent();
+    }
+
+    public void openWithLoginHint(String loginHint) {
+        oauth.addCustomParameter("login_hint", loginHint).openRegistrationForm();
```

Comment:
Minor: Is it possible to use OIDCLoginProtocol.LOGIN_HINT instead of hardcoded constant?